HHS/Head-Start-TTADP
# [TTAHUB-2366] remove communication log feature flag

## Description of change

**This shouldn't be merged in until OHS approves removal**

Remove the feature flag from database and frontend

## How to test

Confirm the feature flag is not present on the users table
Confirm that the feature flag is not needed to view the communication log

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2236


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
